changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md

```diff
@@ -0,0 +1,12 @@
+
+<!--
+Create a changelog entry for every new user-facing change. Please respect the following instructions:
+- Indicate breaking changes by prepending an explosion 💥 character.
+- Prefix your changes with either [Bugfix], [Improvement], [Feature], [Security], [Deprecation].
+- You may optionally append "(by @<author>)" at the end of the line, where "<author>" is either one (just one)
+of your GitHub username, real name or affiliated organization. These affiliations will be displayed in
+the release notes for every release.
+-->
+
+<!-- - 💥[Feature] Foobarize the blorginator. This breaks plugins by renaming the `FOO_DO` filter to `BAR_DO`. (by @regisb) -->
+[Improvement] - Change ecommerce and ecommerce-worker remotes from `edx` to `openedx`. (by @christopappas)
```

Comment:
- [Improvement] Change ecommerce and ecommerce-worker remotes from `edx` to `openedx`. (by @christopappas)

Reply:
I have removed the "-" following [Improvment]

Reply:
"-" will come before [Improvement].